Project/Project/Enums/Actor/Occupation.cs

```diff
@@ -0,0 +1,41 @@
+namespace Project.Enums.Occupation
+{
+    public enum Occupation
+    {
+        Actor,
+        Editor,
+        Cinematographer,
+        FilmDirector,
+        FilmProducer,
+        FilmScoreComposer,
+        Screenwriter,
+        Publisher,
+        TelevisionDirector,
+        TelevisionProducer,
+        Writer,
+        VoiceActor
```

Comment:
What about animator for animation movies like animes etc. ? Please do some more research about possible relevant occupations :)

Reply:
I have searched many animation movies and none of them had these occupations, so in my opinion it is not necessary.

Reply:
https://pixelprime.maxih.de/actor/Hayao%20Miyazaki

Pleae add at least animator